src/shared/templates/shared/partial/sidebar_bottom_end.html

```diff
@@ -5,8 +5,8 @@
             data-clusive-tip-action="tour"
             onclick="tour.chain('#tour_{{ tours|list_index:0 }}');"><span class="sr-only">Start tour of features</span></button>
```

Comment:
Should this be tour.chainStart() ? Clicking the button as is just throws an error for me.

Reply:
Ah, merge error on my part, pushing a fix